### ISSUE TITLE
UI doesn't support exclude files

### ISSUE THREAD

**Issue by songmeo** [argoproj/argo-cd#10519]
# Summary
There is no field to add should-be-excluded files in argoCD UI.

# Motivation
When adding new applications, I should be able to exclude certain files. 

For example, when I want to exclude `blobby-blob.json`, I have to use `edit as YAML` option to add `exclude: blobby-blob.json`

**Comment by crenshaw-dev** [comment]
Yep. Wrote a few more details here: https://github.com/argoproj/argo-cd/issues/10880
